apps/frontend/src/components/RequestModal/RequestModal.tsx

```diff
@@ -18,13 +19,97 @@ interface Props {
 
 export default function RequestModal({ show, handleCloseModal, request }: Props) {
   const [loading, setIsLoading] = useState(false);
+  const [validated, setValidated] = useState(false);
+  const [showForm, setShowForm] = useState(false);
   const revalidator = useRevalidator();
+  const { id: neighborhoodId } = useParams();
+  const submit = useSubmit();
+
+  const user = localStorage.getItem('user');
+  const { username, id: userId } = user ? JSON.parse(user) : null;
+
+  const validateTextArea = (event: FocusEvent<HTMLTextAreaElement>) => {
```

Comment:
We could extract this on a separate utils file since we're reusing it, for now, at least.

Reply:
I didn't extract it to a separate utils module, but I replaced it with the getStoredUser utility function. I should of used that in the first place.
Do you think it's still necessary to extract to a separate utils file now?

Reply:
I was actually referring to the `validateTextArea` function. However, given its shortening, I believe it's trivial to move it at the moment.